demo/server/src/auth/local.ts

```diff
@@ -0,0 +1,10 @@
+import * as express from 'express';
+
+export function loginRequired(req: express.Request, res: express.Response, next: Function) {
```

Comment:
Cool, just checking as the naming was ringing bells with keycloak.

login-required in keycloak just enforces a user to be logged into keycloak. ie. redirect to login if they are not authenticated, and leave them alone if authenticated, it just handles the auth mechanism side of things really.